docs/modules/databases/mongodb.md

```diff
@@ -36,6 +43,90 @@ For instance, to initialize a single node replica set on fixed ports via Docker,
+[Creating a MongoDB Atlas Local Container](../../../modules/mongodb/src/test/java/org/testcontainers/containers/MongoDBAtlasLocalContainerTest.java) inside_block:creatingAtlasLocalContainer
+<!--/codeinclude-->
+
+And how to start it:
+
+<!--codeinclude-->
+[Start the Container](../../../modules/mongodb/src/test/java/org/testcontainers/containers/MongoDBAtlasLocalContainerTest.java) inside_block:startingAtlasLocalContainer
+<!--/codeinclude-->
+
+The connection string provided by the MongoDBAtlasLocalContainer's getConnectionString() method includes the dynamically allocated port:
+
+<!--codeinclude-->
+[Get the Connection String](../../../modules/mongodb/src/test/java/org/testcontainers/containers/MongoDBAtlasLocalContainerTest.java) inside_block:getConnectionStringAtlasLocalContainer
+<!--/codeinclude-->
+
+e.g. `mongodb://localhost:12345`
+
+### General info
+MongoDB Atlas Local combines the MongoDB database engine with MongoT, a sidecar process for advanced searching capabilities built by MongoDB and powered by [Apache Lucene](https://lucene.apache.org/). 
+
+It allows you to use the following features:
+
+* MongoDB Atlas Search: Atlas Search gives MongoDB queries access to the incredible search toolbox that is Lucene. The main use-case is advanced lexical text querying capabilities similar to those found in many search engines. In addition, Atlas Search supports queries with faceting and parallel index search. These can extend MongoDB's aggregation capabilities and performance for uses like statistics and complex filters.  
+  [https://www.mongodb.com/docs/atlas/atlas-search/](https://www.mongodb.com/docs/atlas/atlas-search/)
+* MongoDB Atlas Vector Search: Supports artificial intelligence (AI) based searches for semantically similar items in your data. Vector indexes store embeddings (high-dimensional vectors encoding semantic meaning) used in large language models (LLMs). This feature makes use of Lucene's [vector search](https://lucene.apache.org/core/9_10_0/core/org/apache/lucene/search/KnnVectorQuery.html) capabilities to find the nearness between the values of each vector. This can be a powerful alternative or compliment to lexical text search.  
+  [https://www.mongodb.com/docs/atlas/atlas-vector-search/vector-search-overview/](https://www.mongodb.com/docs/atlas/atlas-vector-search/vector-search-overview/)
+
+Pairing these Lucene backed technologies with your MongoDB database allows you to build powerful search capabilities into your applications without the need to manage a separate search engine. You can also extend your search capabilities to include AI based vector searches, which can be useful for recommendation engines, image search, and other applications that require similarity searches.
+
+The container (mongodb/mongodb-atlas-local) documentation can be found here:
+[https://www.mongodb.com/docs/atlas/cli/current/atlas-cli-deploy-docker/](https://www.mongodb.com/docs/atlas/cli/current/atlas-cli-deploy-docker/)
```

Comment:
I think we can add a link to MongoDB Atlas documentation instead of adding product related information in our docs.

Reply:
Done

Reply:
@eddumelendez should we remove the 'incubating' statement from the top of this docs page? It's been there a long while!
<img width="424" alt="image" src="https://github.com/user-attachments/assets/3fac94ad-eb97-4386-8420-004f8df89139">

Is it still incubating?

Reply:
let's remove it :)